### PR TITLE
Introduce infra/kv and and data.DBServer impl. based on it

### PR DESCRIPTION
**Pull request by nlandolfi**
This moves us toward goal of 0.1, by simplifying our persistence dependency from a mongo cluster to a single file.

### CODE REVIEW

**Comment by nlandolfi** [comment]
Built on top of https://github.com/elos/x/pull/35, could be another git mistake by me. Planning to manage this merge better.

File: jobs/hestia/main.go [hunk 1]
```diff
@@ -125,6 +125,10 @@ func main() {
 
 	mux := http.NewServeMux()
 
+	// health check for k8s ingress
+	mux.HandleFunc("/", logRequest(cors(http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
```
Comment (viggyfresh) [inline jobs/hestia/main.go]: textbook - return 200 to make sure alive

Comment (nlandolfi) [inline jobs/hestia/main.go]: 👍

**Comment by nlandolfi** [comment]
@viggyfresh need approval to merge this, thoughts? depends on https://github.com/elos/x/pull/35

**Pull request merged**